DataConnectors/O365 DataCSharp/Teams.CustomConnector.StorageHandler/StorageHandler.cs

```diff
@@ -94,15 +94,16 @@ public async Task<OperationDetails> GetLastOperationDetailsFromLogsAsync(string
             {
                 log.LogInformation(Common.Constants.RequestLastExecutionTime);
                 CreateCloudStroageHandler();
-                var myClient = storageAccount.CreateCloudBlobClient();
-                var container = myClient.GetContainerReference(Environment.GetEnvironmentVariable(Constants.LogContainerName));
+                var container = blobServiceClient.GetBlobContainerClient(Environment.GetEnvironmentVariable(Constants.LogContainerName));
                 if (await container.ExistsAsync())
                 {
-                    var blockBlob = container.GetBlockBlobReference(blobName);
+                    var blockBlob = container.GetBlockBlobClient(blobName);
                     if (await blockBlob.ExistsAsync())
                     {
-                        var result = await blockBlob.DownloadTextAsync();
-                        operationDetails = Newtonsoft.Json.JsonConvert.DeserializeObject<OperationDetails>(result);
+                        var result = await blockBlob.DownloadAsync();
+                        StreamReader reader = new StreamReader(result.Value.Content);
```

Comment:
`StreamReader` is `IDisposable`
```suggestion
                        using StreamReader reader = new StreamReader(result.Value.Content);
```